Use $^X rather than "perl"

MartinMcGrath opened the pull request:
Thanks for your work on this module.
This short pull request changes the hardcoded "perl" to `$^X`:

https://rt.cpan.org/Public/Bug/Display.html?id=95263

MartinMcGrath committed:
Use $^X rather than "perl"

Thanks for your work on this module.
This short pull request changes the hardcoded "perl" to `$^X`:

https://rt.cpan.org/Public/Bug/Display.html?id=95263

mrscotty commented:
Hi Martin,

Thanks for your input and pull request. I looked at the request and noticed that the travis build with your patch fails. Can you suggest an alternative?

Best regards,

Scott

On May 2, 2014, at 11:33, Martin McGrath notifications@github.com wrote:

> Thanks for your work on this module.
> This short pull request changes the hardcoded "perl" to $^X:
> 
> https://rt.cpan.org/Public/Bug/Display.html?id=95263
> 
> You can merge this Pull Request by running
> 
>   git pull https://github.com/MartinMcGrath/Config-Versioned patch-1
> Or view, comment on, or merge it at:
> 
>   https://github.com/mrscotty/Config-Versioned/pull/3
> 
> Commit Summary
> 
>   • Use $^X rather than "perl"
> File Changes
> 
>   • M t/07-cfgexp.t (2)
> Patch Links:
> 
>   • https://github.com/mrscotty/Config-Versioned/pull/3.patch
>   • https://github.com/mrscotty/Config-Versioned/pull/3.diff
> —
> Reply to this email directly or view it on GitHub.

MartinMcGrath committed:
Use $^X rather than "perl"

Fix interpolation bug of $cfgexp I accidentally introduced.

This short pull request changes the hardcoded "perl" to $^X:

https://rt.cpan.org/Public/Bug/Display.html?id=95263

MartinMcGrath commented:
Hi Scott,

Thanks for bringing this to my attention. I've altered the pull request, resolving the interpolation bug I accidentally introduced. I've tested by building from scratch on a dev server. I'm sorry for the inconvenience this has caused.

mrscotty commented:
Thanks for the patch!!!
